resources/.ktlint_editorconfig

```diff
@@ -4,3 +4,11 @@ ktlint_standard_max-line-length = 160
 ktlint_standard = enabled
 ktlint_standard_multiline-expression-wrapping = disabled
 ktlint_standard_string-template-indent = disabled
+ktlint_standard_chain-method-continuation = disabled
```

Comment:
This seems incorrect (compared to your comment).

Rather it looks like what you need might be: ktlint_chain_method_rule_force_multiline_when_chain_operator_count_greater_or_equal_than

I'm also not 100% convinced this is the right way to go, if IntelliJ would anyway reformat it to multi-line (as the Ktllint docs seems to indicate). They do also have a point that too much chaining obfuscates the code.

Reply:
Intellij doesn't seem to autoformat this, even when asking it to remove custom line-breaks

Reply:
Changed to use the more specific rule. I wish the linter gave better feedback about which specific rule is being broken, rather than just giving the general type..

Reply:
Based on the amount of chain calls required in a bunch of our projects (notably tests in build-conf and test suites, I'm opposed to keeping this rule enabled unless we set the limit very high